app/styles/modules/_password-row.scss

```diff
@@ -15,7 +15,12 @@
   }
 
   .input-help {
-    text-align: left;
+    html[dir=ltr] & {
```

Comment:
Same comment here [about alignment that I made in #3778](https://github.com/mozilla/fxa-content-server/pull/3778#discussion_r64713905)

The closing brace should be left aligned with the selector. a two space indent, and a blank line above the entire declaration:

```css
.input-help {
  html[dir=ltr] & {
    text-align: left;
  }

  html[dir=rtl] & {
    text-align: right;
  }
}
```